travel-journal/index.md

```diff
@@ -0,0 +1,926 @@
+
+In your Vue.js project, navigate to the `src` folder. Delete the `base.css` file and edit the `main.css` file to contain the following: 
+
+```css
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap');
+
+*, body{
+  font-family: 'Poppins', sans-serif;
+}
+```
+
+Doing this will set the overall font for the Vue.js application to [Poppins](https://fonts.google.com/specimen/Poppins).
+
+Head over to the `components` folder and delete everything in it ncluding the `icons` folder and the following components `HelloWorld.vue`, `TheWelcome.vue` and `WelcomeItem.vue`.
+
+Once that is done, go into the `router/index.js` file and edit it to match the following:
+
+```javascript
+import { createRouter, createWebHistory } from 'vue-router'
+import LoginView from '../views/LoginView.vue'
+
+const router = createRouter({
+  history: createWebHistory(import.meta.env.BASE_URL),
+  routes: [
+    {
+      path: '/',
+      name: 'login',
+      component: LoginView
+    },
+    {
+      path: '/signup',
+      name: 'signup',
+      component: () => import('../views/SignupView.vue')
+    },
+    {
+      path: '/addjournal',
+      name: 'addjournal',
+      component: () => import('../views/Addjournal.vue'),
+    },
+    {
+      path: '/readjournal',
+      name: 'readjournal',
+      component: () => import('../views/Readjournal.vue')
+    }
+  ]
+})
+
+export default router
+```
+
+This will set up the basic routing for the Vue.js application, allowing users to navigate between different views. Below is a quick summary of the views:
+
+- `/`: This is the root path of the application and is associated with a view called `LoginView`. When users visit the base URL of your application, they are directed to this view, which is typically used for user authentication.
+- `/signup`: This path leads to the `SignupView` and is designed for new users to register or create an account. When users navigate to `/signup`, they are presented with a signup form to enter their details and create a new account.
+- `/addjournal`: This is the part for the `Addjournal` view. This route is used for adding new journal entries.
+- `/readjournal`: • This route is linked to the `Readjournal` view. It is intended for users to view and read their existing journal entries. When users navigate to `/readjournal`, they can browse through all the journal entries they have created.
+
+In the `views` folder, delete the existing views - `AboutView.vue` and `HomeView.vue` and create the following views - `LoginView.vue`, `SignupView.vue`, `Addjournal.vue` and `Readjournal.vue`.
+
+Finally, edit your `App.vue` to contain the following code:
+
+```html
+<script setup>
+import { RouterView } from 'vue-router'
+
+</script>
+
+<template>
+  <RouterView />
+</template>
+```
+
+## Installing the Directus SDK
+With your Vue.js project set up, proceed to install the Directus SDK with the following command:
+
+```bash
+npm install @directus/sdk
+```
```

Comment:
Install dependencies at the same time as initializing the app.

Reply:
done.